CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 Unreleased Changes
 ------------------
 
+* Feature - Uses `rack` version 3 as the minimum.
+
+* Feature - Drop support for Ruby 2.5 and 2.6.
+
+* Feature - Support additional configuration options through ENV.
+
```

Comment:
Add the TTL Feature?

Reply:
It's in the README - it's nothing in the gem that needs to be in the changelog. It requires manual setup of ttl on the expires_at.